libtomcrypt.pc.in

```diff
@@ -1,10 +1,10 @@
-prefix=/usr
+prefix=@to-be-replaced@
```

Comment:
No objections.

Personally, I’d rather we’d use autotools for consistency with other FOSS projects, but I don’t have the time to work on it.